docs/design/projects/2021_consents.md

```diff
@@ -4,3 +4,13 @@
 
 This project aims to provide more configurable, more extensible and more descriptive
 consents for the users.
+
+## Previous design
+
+Consents were stored in the Person model.
```

Comment:
@pbanaszkiewicz I'm not that familiar with the old design or all the motivations behind the change to consents. Feel free to add a line or two here if you want

Reply:
@elichad updated.